Frameworks/UIKit/ForceInclusion.mm

```diff
@@ -47,6 +48,7 @@ void ForceInclusion() {
     [UISwitch class];
     [UISlider class];
     [UIProxyObject class];
+    [RTObject class];
```

Comment:
@bviglietta  Should they be treated as something else? It doesn't make sense for them to be used as RTObject if they aren't actually one right?